dbms/src/Server/MetricsPrometheus.cpp

```diff
@@ -164,7 +164,13 @@ std::shared_ptr<Poco::Net::HTTPServer> getHTTPServer(
         key_path,
         cert_path,
         ca_path,
-        Poco::Net::Context::VerificationMode::VERIFY_STRICT);
+#if SERVERLESS_PROXY == 0
+        Poco::Net::Context::VerificationMode::VERIFY_STRICT
+#else
+        // mtls: metrics server allows anonymous pullers @iosmanthus
+        Poco::Net::Context::VerificationMode::VERIFY_RELAXED
```

Comment:
Where does this line come from? Can not find on either tiflash master or cse

Reply:
it has been merge months ago https://github.com/tidbcloud/tiflash-cse/pull/216